scripts/ci/install-istio.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+DIR=$(dirname $0)
+
+OS=linux
+TARGET_DIR=/usr/bin
+
+ISTIO_VERSION="1.0.0"
+ISTIO_URL="https://github.com/istio/istio/releases/download/$ISTIO_VERSION/istio-$ISTIO_VERSION-$OS.tar.gz"
+ISTIO_PKG="istio-$ISTIO_VERSION"
+ISTIO_OBJECTS="$TARGET_DIR/$ISTIO_PKG/install/kubernetes/istio-demo.yaml"
+ISTIO_NS=istio-system
+
+uninstall_istio() {
+        sudo rm -rf $TARGET_DIR/$ISTIO_PKG
+        sudo rm -rf $TARGET_DIR/istioctl
+}
+
+install_istio() {
+        curl -L "$ISTIO_URL" | tar xz
+        if [ $? != 0 ]; then
+                echo "failed to download $url"
+                exit 1
+        fi
+
+        chmod a+x $ISTIO_PKG
+        sudo cp $ISTIO_PKG/bin/istioctl $TARGET_DIR/.
+        sudo mv $ISTIO_PKG $TARGET_DIR/.
```

Comment:
don't pollute /usr/bin
install the istio-1.0.1 directory to /usr/share/istio

Reply:
Minikube and Kubectl are located in the same place; 
We want the installation be accessible by PATH...